eoscompanion/ekn_data.py

```diff
@@ -31,6 +31,22 @@
 LOAD_FROM_ENGINE_NO_SUCH_CONTENT = 2
 
 
+def load_record_blob_from_shards(shards, content_id, attr):
+    '''Load a blog for an app and content_id if you already have the shards.'''
```

Comment:
Typo: `blog`. Also (nitpick) I'd probably not use the second person in the docstring. (E.g. `Load a blob for an app and content_id given a set of shards.`)

Reply:
Agreed